services/incentive/src/test/java/org/cryptimeleon/incentive/services/incentive/ClientHelper.java

```diff
@@ -110,6 +110,7 @@ static EarnRequest generateAndSendEarnRequest(WebTestClient webTestClient,
         webTestClient.post()
                 .uri("/bulk-token-updates")
                 .header("basket-id", String.valueOf(basketId))
+                .header("do-sync", Boolean.toString(false))
```

Comment:
What does this do?

Reply:
I am glad that it does nothing because it is something from the last attempt on this feature that I forgot to remove.
Thanks for pointing this out ^^